docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,254 @@
+
+Installation
+============
+
+For the first step of our journey, let's install MCDR
+
+.. note::
+
+    We assume you have some really basic knowledge of Python and pip.
+
+    If you have not, Google is your friend
+
+Requirements
+------------
+
+.. image:: https://img.shields.io/pypi/pyversions/mcdreforged.svg
+   :alt: Python version
+
+.. image:: https://img.shields.io/pypi/v/mcdreforged.svg
+   :alt: PyPI version
+
+MCDR is written and runs in Python 3. Make sure you have Python 3.8 or later with pip3 installed
+
+The detailed Python version requirements are as shown in the table below
+
+.. list-table::
+   :header-rows: 1
+
+   * - MCDR version
+     - Python requirement
+   * - < 2.10
+     - >= 3.6
+   * - >= 2.10
+     - >= 3.8
+
+Wait a second
+-------------
+
+In some "tutorials", you may learned to use MCDR by downloading a zip from Github and extract it
+
+However, that's not the actual way to install MCDR - Those tutorials are outdated
+
+.. caution::
+
+    **DO NOT** download the source files of MCDR and execute them directly, unless you're a developer of MCDR and you know what you are doing
+
+Install using pip
+-----------------
+
+MCDR is available in `PyPI <https://pypi.org/project/mcdreforged>`__, it's reasonably to be installed by:
+
+.. tab:: Windows
+
+    .. code-block:: bat
+
+        pip install mcdreforged
+
+.. tab:: Linux
+
+    .. code-block:: bash
+
+        pip3 install mcdreforged
+
+And upgraded by:
+
+.. tab:: Windows
+
+    .. code-block:: bat
+
+        pip install mcdreforged -U
+
+.. tab:: Linux
+
+    .. code-block:: bash
+
+        pip3 install mcdreforged -U
+
+Verify the installation with:
+
+.. prompt-mcdr-version:: bash $ auto
+
+    $ mcdreforged
+    MCDReforged v@@MCDR_VERSION@@
+
+Externally managed environment
+------------------------------
+
+If you're using Windows, the command above should work fine, MCDR will be installed to global environment - you may ignore this section
+
+For Linux and Mac OS, it's not recommended to install MCDR system-wide, because it can cause conflicts with other Python packages and affect system dependencies
```

Comment:
User might not know what "system-wide" is. You can say "use root to install MCDR"